### PR TITLE
Add description metadata

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/IBX-1440
| Versions      | all

### CODE REVIEW

File: docs/tutorials/platform_beginner/8_enable_account_registration.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+---
+description: Se how you can enable external users to register and contribute to your site.
```
Comment (MagdalenaZuba) [inline docs/tutorials/platform_beginner/8_enable_account_registration.md:2]:
```suggestion
description: See how you can enable external users to register and contribute to your site.
```

File: docs/updating/from_1.x_2.x/update_from_1.x_2.x.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 ---
+description: Update your installation to the latest v2.5 version from v1.13 or earlier v2 versions.
```
Comment (MagdalenaZuba) [inline docs/updating/from_1.x_2.x/update_from_1.x_2.x.md:2]:
```suggestion
description: Update your installation to the latest v2.5 version from v1.13 or earlier v2 version.
```

**Review by MagdalenaZuba — CHANGES_REQUESTED** [review]

File: docs/community_resources/release_process.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+---
+description: "Ibexa DXP releases new versions periodically in three flavors: Ibexa Content, Ibexa Experience and Ibexa Commerce, plus open-source Ibexa OSS."
```
Comment (DominikaK) [inline docs/community_resources/release_process.md:2]:
```suggestion
description: "Ibexa DXP releases new versions periodically in different flavors: Ibexa Content, Ibexa Experience and Ibexa Commerce, plus open-source Ibexa OSS."
```
Forgot to add in PR decription: quotation marks are required when description contains a colon, otherwite the yaml formatting is broken

File: docs/guide/content_model.md [hunk 4]
```diff
@@ -1,3 +1,7 @@
+---
+description: Ibexa DXP's content model relies on Content items based on Content Types, containing different content Fields.
```
Comment (DominikaK) [inline docs/guide/content_model.md:2]:
```suggestion
description: Ibexa DXP's content model relies on Content items that are instances of Content Types and contain content Fields.
```

File: docs/guide/search/search.md [hunk 5]
```diff
@@ -1,3 +1,7 @@
+---
+description: Ibexa DXP search functionalities allows working with three search engines and using search API to run complex and precise question of content and products.
```
Comment (DominikaK) [inline docs/guide/search/search.md:2]:
```suggestion
description: Ibexa DXP search functionalities allow working with three search engines and using search API to run complex and precise queries about content and products.
```
should have been 'queries' from the start, not sure how 'questions' got there

**Review by MagdalenaZuba — APPROVED** [review]

**Pull request merged**